internal/entities.go

```diff
@@ -83,55 +84,72 @@ func setupEntity(ctx context.Context, doc *enigma.Doc, entityPath string, entity
 	if err != nil {
 		FatalError("Could not open "+entityType+" file", err)
 	}
-	var props genericEntity
-	err = json.Unmarshal(entityFileContents, &props)
-	validateEntity(props, entityPath, err)
-
+	var entity genericEntity
+	err = json.Unmarshal(entityFileContents, &entity)
+	validateEntity(entity, entityPath, err)
 	//I do not know how to to this nicer, with less duplication.
 	switch entityType {
 	case "dimension":
-		dimension, err := doc.GetDimension(ctx, props.Info.Id)
+		dimension, err := doc.GetDimension(ctx, entity.Info.Id)
 		if err == nil && dimension.Handle != 0 {
-			LogVerbose("Updating dimension " + props.Info.Id)
+			LogVerbose("Updating dimension " + entity.Info.Id)
 			err = dimension.SetPropertiesRaw(ctx, entityFileContents)
 			if err != nil {
-				FatalError("Failed to update dimension "+props.Info.Id, err)
+				FatalError("Failed to update dimension "+entity.Info.Id, err)
```

Comment:
Think we should be consistent with spaces when concatenating strings. Since we are updating the logging messages we might as well fix that now.